src/main/kotlin/com/sourcegraph/cody/context/RemoteRepoUtils.kt

```diff
@@ -27,7 +27,7 @@ object RemoteRepoUtils {
             Graphql_GetRepoIdsParams(codebaseNames.map { it.value }, codebaseNames.size.toLong())
         val repos = agent.server.graphql_getRepoIds(param).get()
         result.complete(
-            repos?.repos?.map { reposParams -> Repo(reposParams.id, reposParams.name) }
+            repos?.repos?.map { reposParams -> Repo(name = reposParams.name, id = reposParams.id) }
```

Comment:
@RXminuS 👋

Reply:
I was trying to get https://github.com/sourcegraph/jetbrains/pull/1930 working - happily, it turned out that my single test for chat is testing something and I was able to find this one 😅

Reply:
nothing, just pinging you as you are the author 😃

Reply:
@RXminuS this is actually an issue caused by introducing generated protocol files.
Parameters order was switched in a old and new class, but since both types were `String` we never realised it's a bug.
But it breaks the logic.

That is something which may happen again in future and it's hard to guard agains unless we use named params everywhere. So just something to be cautious about :)